docs/conf.py

```diff
@@ -32,14 +32,85 @@
 # ones.
 extensions = [
     'sphinx.ext.ifconfig',
-    'breathe'
+    'breathe',
+    'exhale'
 ]
 
+# Setup the `breathe` extension
 breathe_projects = { "NanoGUI": "./doxyoutput/xml" }
 breathe_default_project = "NanoGUI"
 
+# Setup the `exhale` extension
+exhale_args = {
+    ############################################################################
+    # These arguments are required.                                            #
+    ############################################################################
+    "containmentFolder":     "./api",
+    "rootFileName":          "library_root.rst",
+    "rootFileTitle":         "Library API",
+    "doxygenStripFromPath":  "../include",
+    ############################################################################
+    # Suggested optional arguments.                                            #
+    ############################################################################
+    "createTreeView":        True,
+    "exhaleExecutesDoxygen": True,
+    "exhaleDoxygenStdin": textwrap.dedent('''
+        # Tell Doxygen where the source code is (yours may be different).
+        INPUT                  = ../include
+        # Doxygen chokes on `NAMESPACE_BEGIN`, predfine all of these
+        PREDEFINED            += NAMESPACE_BEGIN(nanogui)="namespace nanogui {"
+        PREDEFINED            += NAMESPACE_END(nanogui)="}"
+        PREDEFINED            += NAMESPACE_BEGIN(detail)="namespace detail {"
+        PREDEFINED            += NAMESPACE_END(detail)="}"
+        PREDEFINED            += DOXYGEN_SHOULD_SKIP_THIS
+        PREDEFINED            += DOXYGEN_DOCUMENTATION_BUILD
+        PREDEFINED            += NANOGUI_EXPORT
+    '''),
+    ############################################################################
+    # HTML Theme specific configurations.                                      #
+    ############################################################################
+    # Fix broken Sphinx RTD Theme 'Edit on GitHub' links
+    # Search for 'Edit on GitHub' on the FAQ:
+    #     http://exhale.readthedocs.io/en/latest/faq.html
+    "pageLevelConfigMeta": ":github_url: https://github.com/wjakob/nanogui",
```

Comment:
This is the (temporary, but not going to be fixed any time soon) hack to get the generated pages to link back to a real URL.